web/src/user_groups.ts

```diff
@@ -332,6 +332,12 @@ export function is_subgroup_of_target_group(target_group_id: number, subgroup_id
     return recursive_subgroup_ids.has(subgroup_id);
 }
 
+export function get_user_groups_using_group_as_subgroup(group_id: number): UserGroup[] {
```

Comment:
Do we also want to use the term supergroup here? that seems easier to read to me